Bit-Manipulation/LogTwo.js

```diff
@@ -0,0 +1,19 @@
+/*
+  https://handwiki.org/wiki/Binary_logarithm
+
+  Approximate log2 using only bitwise operators
+*/
+
+/*
```

Comment:
JSDoc requires a bit more I believe. Also omit the blank line above the func, and merge the two comments.

Reply:
See https://jsdoc.app/about-getting-started.html